shared_kv_storage/kv_storage/remote_lmdb/shared_kv_dbstore.py

```diff
@@ -82,9 +82,10 @@ def set(self, table, key, value):
         try:
             self._db_store.db_store_put(table, key, value)
             return True
-        except Exception as ex:
-            logger.error("Exception occurred in set - {}".
-                         format(ex))
+        except Exception:
```

Comment:
In what case db_store_put will throw an error? Add that in a comment as well.

Reply:
@Ram-srini, added TODO for now. Need to put up a full fledged error handling here. As of now several error situation are handled in this blanket handle. The varied errors could be - failing to open database in the environment, failing to create a transaction, failing to run the transaction or if there is a size mismatch between the size of value retrieved/to be inserted and the size calculated.
Will take up the changes in a separate PR to create new error codes and pass them back to caller for them to handle appropriately.

Reply:
Good add error code check in caller side/throw exception in case set fails.